docs/build/build-transaction-construction.md

```diff
@@ -35,6 +35,29 @@ function from the Balances pallet will take:
 - `dest`: Destination address
 - `#[compact] value`: Number of tokens (compact encoding)
 
+**Serialized transaction format**
+
+Before being submitted, transactions are serialized. Serialized transactions are hex encoded SCALE-encoded bytes. The specific serialization is defined in the runtime and can change if the runtime is upgraded, but in general the serialization format can be described as follows:
+- Compact encoded number of SCALE-encoded bytes following this.
+- 1 bit: it is a 0 if no signature is present, or a 1 if it is.
+- 7 bits: the transaction version.
+- If there is a signature:
+  - a SCALE encoded `sp_runtime::MultiAddress::Id<AccountId32, u32>` indicating the signer(s) of the transaction.
+  - a SCALE encoded `sp_runtime::MultiSignature::{SigningScheme}` with the signature\*.
+  - a SCALE encoded `sp_runtime::generic::Era` indicationg for how long this transaction will live in the pool.
```

Comment:
Its being kicked from the pool is just a byproduct of it being invalid from the runtime's perspective (the pool won't include transactions that will _obviously_ fail, e.g. nonce too low, too old, etc.)
```suggestion
  - a SCALE encoded `sp_runtime::generic::Era` indicating for how long this transaction is valid.
```

Reply:
@IkerAlus

Reply:
@DannyS03 - he's on vacation, he'll get back to it when he comes back.

Reply:
good point, changed as suggrested